app_dart/lib/src/request_handlers/presubmit_luci_subscription.dart

```diff
@@ -121,6 +127,29 @@ final class PresubmitLuciSubscription extends SubscriptionHandler {
       slug: userData.commit.slug,
       rescheduled: rescheduled,
     );
+
+    if (!rescheduled && config.flags.closeMqGuardAfterPresubmit) {
+      // Process to the check-run status in the merge queue document during
+      // the LUCI callback.
+      final conclusion = _githubChecksService.conclusionForResult(build.status);
+      await _scheduler.processCheckRunCompleted(
+        cocoon_checks.CheckRun(
+          id: userData.checkRunId,
+          name: builderName,
+          headSha: userData.commit.sha,
+          conclusion: '$conclusion',
+          checkSuite: CheckSuite(
+            id: userData.checkSuiteId,
```

Comment:
Just note that `check_suite_id` can be null when the flag is first flipped; there will be outstanding checkruns for ~50 minutes depending on the time of day we flip the flag.

I _think_ that will still work as the update checkrun REST endpoint doesn't appear to require the check_suite_id:
https://docs.github.com/en/rest/checks/runs?apiVersion=2022-11-28#update-a-check-run